internal/resources/cluster/resource_cluster.go

```diff
@@ -346,12 +358,25 @@ func resourceClusterCreate(ctx context.Context, d *schema.ResourceData, m interf
 			Summary:  "Kubernetes cluster's kubeconfig provided. Proceeding to attach the cluster TMC",
 		})
 
-		deploymentManifests, err := manifest.GetK8sManifest(clusterResponse.Cluster.Status.InstallerLink)
+		if clusterResponse.Cluster.Spec.ImageRegistry != "" {
```

Comment:
are we adding changes accordingly to the acceptance test also?

Reply:
@shobha2626 - As Shubhang mentioned ensure to add acceptance test

Reply:
Done

Reply:
can you please run the acceptance test and attached the screenshot, just make sure the tests are working fine

Reply:
Sure, attached in the description